src/p5.dance.js

```diff
@@ -356,42 +356,49 @@ module.exports = class DanceParty {
 
 // Dance Moves
 
-  changeMoveLR(sprite, move, dir) {
-    if (!this.spriteExists_(sprite)) {
-      return;
-    }
+  getNewChangeMove(requested_change, current_move) {
     // Number of valid full length moves
     const { fullLengthMoveCount, restMoveCount } = this.world;
     const firstNonRestingMoveIndex = restMoveCount;
     // The "rest" moves are assumed to always be at the beginning
     const nonRestingFullLengthMoveCount = fullLengthMoveCount - restMoveCount;
+    if (nonRestingFullLengthMoveCount <= 1) {
+      throw "next/prev/rand requires that we have 2 or more non-resting full length moves";
```

Comment:
in general, it's better practice to throw errors rather than strings, i.e. `throw new Error('next/prev/rand...');`.

I think this is in part because a lot of error handlers expect to get an error object instead. I think it might also mean better call stack reporting, but I might be wrong about that.